File: app/src/test/java/com/fractalwrench/crazycats/common/TextUtilsTest.java
package com.fractalwrench.crazycats.common;

import org.junit.Test;

import static junit.framework.Assert.assertTrue;
import static org.junit.Assert.assertFalse;

public class TextUtilsTest {

    @Test
    public void isEmpty() throws Exception {
        assertTrue(TextUtils.isEmpty(null));
        assertTrue(TextUtils.isEmpty(""));
        assertFalse(TextUtils.isEmpty("test"));
    }

}

File: app/src/test/java/com/fractalwrench/crazycats/common/TextUtilsTest.kt
package com.fractalwrench.crazycats.common

import org.junit.Test

import junit.framework.Assert.assertTrue
import org.junit.Assert.assertFalse

class TextUtilsTest {

    @Test
    @Throws(Exception::class)
    fun isEmpty() {
        assertTrue(TextUtils.isEmpty(null))
        assertTrue(TextUtils.isEmpty(""))
        assertFalse(TextUtils.isEmpty("test"))
    }

}